README.md

```diff
@@ -1217,6 +1217,56 @@ const MyElement = (props, chren) => <div>Hello, {chren()}!</div>;
 
 
 
+#### `vdom.elements (...elements)`
+
+Returns an array of functions that create elements.
+
+```js
+const [ div, p ] = skate.vdom.elements('div', 'p');
+div(p.bind('test'));
```

Comment:
```
div(p('test'), p('test'));
```

Yes.

> But for performance, wouldn't it be better to have a single function like in the first example?

I'm skeptical it would matter enough to warrant a poorer API. I'd want to see numbers on the perf implications before deciding against it.